src/Listener/SaveSuspensionToDatabase.php

```diff
@@ -20,6 +21,21 @@ class SaveSuspensionToDatabase
     use AssertPermissionTrait;
 
     /**
+     * @var SuspendValidator
+     */
+    protected $validator;
+
+    /**
+     * SaveSuspensionToDatabase constructor.
```

Comment:
I don't like these kind of doc strings. They're pointless. If we have nothing to say, we should rather leave them away. :)